plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -237,7 +236,7 @@ func (c *MaxPDVolumeCountChecker) filterVolumes(volumes []v1.Volume, namespace s
 			if err != nil {
 				// if the PVC is not found, log the error and count the PV towards the PV limit
 				// generate a random volume ID since its required for de-dup
-				utilruntime.HandleError(fmt.Errorf("Unable to look up PVC info for %s/%s, assuming PVC matches predicate when counting limits: %v", namespace, pvcName, err))
+				glog.V(4).Infof("Unable to look up PVC info for %s/%s, assuming PVC matches predicate when counting limits: %v", namespace, pvcName, err)
```

Comment:
HandleError has rate limiting of max one error per mili-sec. It also shows call-stack. I am not so sure how all those will be useful, but at least converting an error log to an info log does not seem right to me.

Reply:
I don't see why you would warn in this case, it looks like spam for the control flow.